client/reader/start/main.jsx

```diff
@@ -55,8 +57,16 @@ const Start = React.createClass( {
 		page.redirect( '/' );
 	},
 
+	renderLoadingPlaceholders() {
+		const count = 4;
```

Comment:
We probably should be though. I have always done that in other projects. Worth raising at a framework level?